templates/zerver/api/get-attachments.md

```diff
@@ -0,0 +1,44 @@
+# Get attachments
+
+Get the attachments belonging to the requesting user.
+
+`GET {{ api_url }}/v1/attachments`
```

Comment:
Can you use the new system for getting descriptions from `zerver/openapi/zulip.yaml` here?